app/backend/src/couchers/models/notifications.py

```diff
@@ -297,3 +297,50 @@ class PushNotificationDeliveryAttempt(Base):
     response: Mapped[str | None] = mapped_column(String, nullable=True)
 
     push_notification_subscription = relationship("PushNotificationSubscription")
+
+
+class MobilePushNotificationSubscription(Base):
+    """
+    Mobile push notification subscriptions (Expo/FCM/APNs)
+    """
+
+    __tablename__ = "mobile_push_notification_subscriptions"
+
+    id: Mapped[int] = mapped_column(BigInteger, primary_key=True)
+    created: Mapped[datetime] = mapped_column(DateTime(timezone=True), server_default=func.now())
+
+    user_id: Mapped[int] = mapped_column(ForeignKey("users.id"), index=True)
+
+    token: Mapped[str] = mapped_column(String, unique=True, index=True)
+    platform: Mapped[str] = mapped_column(String)  # "expo", "fcm", "apns"
+    device_name: Mapped[str | None] = mapped_column(String, nullable=True)
+    device_type: Mapped[str | None] = mapped_column(String, nullable=True)  # "ios", "android"
+
+    disabled_at: Mapped[datetime] = mapped_column(DateTime(timezone=True), server_default=DATETIME_INFINITY.isoformat())
+
+    user = relationship("User")
+
+    __table_args__ = (
+        Index(
+            "ix_mobile_push_notification_subscriptions_active",
+            user_id,
+            disabled_at,
```

Comment:
Do we really need to include the 2nd field in the index? A user is unlickely to have more than a dozen subscriptions, so the benefits are minimal

Reply:
Yeah good point, I'll remove it.